commons/src/main/java/io/hyscale/commons/models/ClusterVersionInfo.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.models;
+
+import io.hyscale.commons.constants.ToolConstants;
+
+public class ClusterVersionInfo {
+
+    private String majorVersion;
```

Comment:
Good one !
can we just refer major & minor...version is inferred from the model name I feel.